### ISSUE TITLE
OSS Backup and Restore

### ISSUE THREAD

**Issue by e-dard** [influxdata/influxdb#15417]
Before we can move into a beta phase with InfluxDB 2.0 we need to provide a suitable backup and restore solution. At a minimum we need to provide users with the ability to:

- execute an online backup of their data;
- enable the user to do an offline restore of their data.

In this case _online_ means that the backup is taken whilst the database is fully operational; it should not cause any downtime issues. _offline_ means that it is acceptable to execute this operation whilst the database is not running.

In this case _data_ means:

 - all writes written into the system up to the point that the backup is executed.
 - any meta data needed to restore the database to a condition where all buckets can be read and written into.

It's my understanding that data like "dashboards" and so on are out-of-scope for this feature. We should get clarity on that from product.

For inspiration on the UX of this feature consider the 1.x B+R docs [here](https://docs.influxdata.com/influxdb/v1.7/administration/backup_and_restore/).

- [x] RFC influxdata/idpe#4998
- [x] backup data #15603 
- [x] backup meta data #15604 
- [x] restore data #15605 
- [x] restore meta data #15606

**Comment by jacobmarble** [comment]
Implemented by #16504
